.config/nextest.toml

```diff
@@ -11,13 +11,29 @@ slow-timeout = { period = "10s", terminate-after = 3 }
 # On CI, we use the 'ci' profile, which runs all tests.
 default-filter = "not test(no_aws_credentials)"
 
-[profile.ci]
```

Comment:
can you add a comment explaining how to set the profile in the CLI? Would be helpful for future reference.